var/spack/repos/builtin/packages/py-morph-tool/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMorphTool(PythonPackage):
+    """Python morphology manipulation toolkit"""
+
+    homepage = "https://github.com/BlueBrain/morph-tool"
+    git = "https://github.com/BlueBrain/morph-tool.git"
+    pypi = "morph-tool/morph-tool-2.9.1.tar.gz"
+
+    version("master", branch="master")
+    version("2.9.1", sha256="305e9456c8047726588b23dfa070eb95ccbe5573e9fea3e0a83dc93eacdf61dc")
+    version("2.9.0", sha256="c60d4010e17ddcc3f53c864c374fffee05713c8f8fd2ba4eed7706041ce1fa47")
+
+    variant("neuron", default=False, description="Enable additional neuron support")
+    variant("plotly", default=False, description="Enable additional plotly support")
+    variant("parallel", default=False, description="Enable additional parallel support")
+
+    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools-scm", type=("build", "run"))
+
+    depends_on("py-click@6.7:", type=("build", "run"))
+    depends_on("py-deprecation@2.1.0:", type=("build", "run"))
+    depends_on("py-more-itertools@8.6.0:", type=("build", "run"))
+    depends_on("py-morphio@3,develop", type=("build", "run"))
+    depends_on("py-neurom@3,develop", type=("build", "run"))
```

Comment:
Surprised that develop works but not 4

Reply:
I'm guessing that was a leftover from ancient development times - I'll remove the `develop` dependencies here; if necessary we'll patch them in again in our internal setup.